ingestion/src/metadata/ingestion/source/database/databricks/client.py

```diff
@@ -142,11 +142,11 @@ def list_jobs(self) -> List[dict]:
                 timeout=10,
             ).json()
 
-            job_list.extend(response["jobs"])
+            job_list.extend(response.get("jobs"))
```

Comment:
@MilanBariya in case the `response.get` returns `None` it better to add `esponse.get("jobs") or []`